stdlib/REPL/test/replcompletions.jl

```diff
@@ -2484,3 +2502,109 @@ let (c, r, res) = test_complete_context("global xxx::Number = Base.", Main)
     @test res
     @test "pi" ∈ c
 end
+
+# #55842
+let (c, r) = test_complete_pos("@tim| using Date")
+    @test "@time" in c
+    @test r == 1:4
+end
+
+# #56389
+let s = "begin\n  using Linear"
+    c, r = test_complete(s)
+    @test "LinearAlgebra" in c
+    @test r == 15:20
+    @test s[r] == "Linear"
+end
```

Comment:
This doesn't pass when run via `Pkg.test("REPL")` because `Base.runtests` puts all stdlibs on the load path, whereas `Pkg.test` only does REPL deps and test deps.
```suggestion
let s = "begin\n  using Ran"
    c, r = test_complete(s)
    @test "Random" in c
    @test r == 15:17
    @test s[r] == "Ran"
end
```